app/models/manageiq/providers/vmware/infra_manager/provision/configuration/disk.rb

```diff
@@ -21,6 +21,37 @@ def build_config_disk_spec(vmcs)
     end
   end
 
+  def build_disk_relocate_spec(_clone_options)
+    disks_info.map do |_disk|
+      VimHash.new("VirtualDiskFlatVer2BackingInfo") do |vdbi|
+        case get_option(:disk_format)
+        when 'thin'
+          vdbi.thinProvisioned = true # VimString.new('sparse', "VirtualMachineRelocateTransformation")
+        when 'thick'
+          vdbi.thinProvisioned = false
+          vdbi.eagerlyScrub = false # then VimString.new('flat', "VirtualMachineRelocateTransformation")
+        when 'thick_eager'
+          vdbi.eagerlyScrub = true
+          vdbi.thinProvisioned = false
+        end
+      end
+    end
+  end
+
+  def disks_info
+    inventory_hash = source.with_provider_connection do |vim|
+      vim.virtualMachineByMor(source.ems_ref_obj)
+    end
+
+    devs = inventory_hash.fetch_path("config", "hardware", "device") || []
+    devs.each_with_object([]) do |dev, disks|
+      next unless dev.fetch_path("deviceInfo", "label").to_s =~ /^Hard\sdisk\s.*$/
```

Comment:
INFO -- : XXX2 adding {"key"=>"100", "deviceInfo"=>{"label"=>"PCI controller 0", "summary"=>"PCI controller 0"}, "busNumber"=>"0", "device"=>["500", "12000", "1000", "4000"]} to disks and PCI controller 0

Welp, I don't think that's a hard disk really at all... I don't know if this line is working, where it appears in this file in the code as it is.